packages/flame_tiled/lib/src/renderable_tile_map.dart

```diff
@@ -49,11 +57,20 @@ class RenderableTiledMap {
     Vector2 destTileSize,
   ) async {
     final map = await _loadMap(contents);
-    final batches = await _loadImages(map);
-
-    return RenderableTiledMap(map, batches, destTileSize);
+    final batchesByLayer = await Future.wait(
+      _renderableTileLayers(map).map((e) => _loadImages(map)),
+    );
+
+    return RenderableTiledMap(
+      map,
+      batchesByLayer,
+      destTileSize,
+    );
   }
 
+  static Iterable<TileLayer> _renderableTileLayers(TiledMap map) =>
+      map.layers.where((layer) => layer.visible).whereType<TileLayer>();
```

Comment:
All functions that spread across more than one line in our code base should use the `{ }` notation.